alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -76,6 +66,7 @@ http {
 
   more_set_headers "X-Content-Type-Options: nosniff";
   more_set_headers "X-Frame-Options: deny";
+  more_set_headers "Content-Security-Policy: frame-ancestors 'none'"
```

Comment:
I think we can go even more strict by default and prevent any JS literally present in HTML documents. This will knock out classical XSS.